data/autoyast_sle15/pc_tools.xml

```diff
@@ -116,13 +116,14 @@
       <partitions config:type="list">
         <partition>
           <mountby config:type="symbol">device</mountby>
-          <filesystem config:type="symbol">swap</filesystem>
-          <mount>swap</mount>
+          <filesystem config:type="symbol">ext4</filesystem>
+          <mount>/</mount>
         </partition>
         <partition>
           <mountby config:type="symbol">device</mountby>
-          <filesystem config:type="symbol">ext4</filesystem>
-          <mount>/</mount>
+          <filesystem config:type="symbol">swap</filesystem>
+          <mount>swap</mount>
+          <size>2G</size>
```

Comment:
I would get rid of the swap completely.

Reply:
Good idea. I would still proceed with this PR to first reduce the swap space and then remove it, so we can easier roll back if needed.

Let's do two steps instead of one.

Reply:
https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/15417